middlewares/members.js

```diff
@@ -5,7 +5,7 @@ const constants = require('../lib/constants');
 const helpers = require('../lib/helpers');
 const errors = require('../lib/errors');
 const mailer = require('../lib/mailer');
-const { sequelize } = require('../lib/sequelize');
+const { sequelize, Sequelize } = require('../lib/sequelize');
```

Comment:
Can't we use one of the two consistently?

Reply:
these are 2 different things unfortunately